src/utils/colors.ts

```diff
@@ -24,15 +26,125 @@ export const colors = {
     gray700: '#455a64',
     gray800: '#37474f',
     gray900: '#263238',
-    red: '#f44336',
-    indigo: '#3f51b5',
-    amber: '#ffc107',
-    purple: '#9c27b0',
-    green: '#4caf50',
-    deepPurple: '#673ab7',
-    orange: '#ff9800',
-    lime: '#cddc39',
-    pink: '#e91e63',
-    blueSlateLighter: '#7c98d3',
-    greenLighter: '#b1cbb3',
```

Comment:
Not sure if removing default colors won't break anything.

Reply:
If not used that's ok

Reply:
We can't have both scales and the ones I deleted